docs/conf.py

```diff
@@ -60,7 +60,7 @@
 # -- Extension options -------------------------------------------------------
 
 # This allows us to use ::: to denote directives, useful for admonitions
-myst_enable_extensions = ["colon_fence", "substitution"]
+myst_enable_extensions = ["colon_fence", "linkify", "substitution"]
```

Comment:
That's why the links were not directly working in myst files ?

Reply:
yes correct